### ISSUE TITLE
Add documentation for ADC API requirements

### ISSUE THREAD

**Issue by bcorrie** [airr-community/airr-standards#597]
Reflect the following as discussed at CRWG and Standards:

- TSV output SHOULD be supported for all data that is commonly represented in tabular structures
- All API endpoints SHOULD return JSON encodings
- For some API endpoints it is possible to request TSV files
- For any endpoints supporting TSV encodings these will be clearly documented in the API Spec and/or Documentation
- Endpoints that do not support TSV can reject TSV requests
- If API endpoint returns a field then the content of that field in JSON and TSV MUST be equivalent
- Bottom line: API Endpoints that are JSON or JSON+TSV should be defined in the spec/docs - the repositories need to follow this spec.

**Comment by bcorrie** [comment]
Need to clarify the wording for:

"TSV output SHOULD be supported for all data that is commonly represented in tabular structures"

This is more a file formats standards question, is that what we want to say?
